src/main/java/com/minecolonies/commands/ColoniesCommand.java

```diff
@@ -0,0 +1,31 @@
+package com.minecolonies.commands;
+
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+/**
+ * Minecolonies root command.
+ * <p>
+ * Manages all sub commands.
+ */
+public class ColoniesCommand extends SplitCommand
+{
+
+    public ColoniesCommand()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)